docs/src/hugo/layouts/index.html

```diff
@@ -0,0 +1,146 @@
+                <p>
+                  <a class="btn btn-primary btn-lg" href="/introduction/" role="button">Getting Started</a>
+                  &nbsp;
+                  <a class="btn btn-light btn-lg"
+                     href="https://hub.docker.com/r/getnelson/nelson/tags/"
+                     target="_blank"
+                     role="button">Download
+                  {{ with .Site.Data.global }}
+                    {{ .release_version }}
+                  {{ else }}
+                    {{ $.Site.Params.version }}
+                  {{ end }}</a>
+                </p>
+              </div>
+
+            </div>
+          </div>
+        </div>
+      </div> <!-- /jumbotron -->
+
+      <div class="jumbotron jumbosplash bg-blue-5">
+        <div class="container">
+          <div class="row">
+            <div class="col-md-12">
+              <h2 class="semibold">Open-source, Container-first Orchestration</h2>
+              <p class="lead">
+                Nelson is a cutting edge orchestration engine that empowers users to focus on building their products, whilst providing operators with the safety and confidence user applications are meeting the organizationl best practices. Having its roots in type-theory and academic research, Nelson fuses field experience with formal rigor to make deployments something that "just works".
+              </p>
+            </div>
+          </div>
+          <div class="row">
+            <div class="col-md-6">
+              <h2>Immutable Deployments</h2>
+              <p>Nelson takes immutable infrastructure to the next level: each and every deployment is fixed for its lifetime, and upgrades are conducted by migrating traffic between versions. This enables powerful production-testing and experimentation.</p>
+              <!-- <p><a class="btn btn-secondary" href="#" role="button">View details &raquo;</a></p> -->
+            </div>
+            <div class="col-md-6">
+              <h2>Scheduler Agnostic</h2>
+              <p>Users face a plethora of choices when it comes to picking a scheduler - you may even be running multiple kinds of scheduler. Nelson enables operators to focus on scheduling (without lossing power), whilst users focus on shipping their product.</p>
+            </div>
+            <div class="col-md-6" style="padding-top: 2em;">
+              <h2>Git-centric</h2>
+              <p>Out of the box integration with Github (SaaS and Enterprise versions) means Nelson already integrates into the workflow you know and love. Simply create a project release on Github and Nelson will distribute and run your program container.</p>
+            </div>
+            <div class="col-md-6" style="padding-top: 2em;">
+              <h2>Secure By Default</h2>
+              <p>The easy thing should also be the safe thing. Nelson has security baked-in: every stack launched with Nelson has unique Vault policies, with ephemeral, rotating credentials that are unique per-container.</p>
```

Comment:
Perhaps a weaker statement in case folks coming across this page aren't/don't want to run Vault. Maybe something like "Security Integrations" and "Nelson supports Vault policies.." as opposed to it being explicitly baked -in.